web/react/utils/utils.jsx

```diff
@@ -95,9 +96,8 @@ module.exports.getCookie = function(name) {
   if (parts.length == 2) return parts.pop().split(";").shift();
 }
 
-
 module.exports.notifyMe = function(title, body, channel) {
-  if ("Notification" in window && Notification.permission !== 'denied') {
+    if ("Notification" in window && Notification.permission !== 'denied') {
```

Comment:
Yeah, I originally had made some changes to that function and must have indented that line because it bugged me :P I'll go ahead and do some more cosmetic refactoring to that function since I already started doing it.